book.rb

```diff
@@ -0,0 +1,17 @@
+require './rental'
+
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
```

Comment:
- [ ] please the method for addint rental should take in the person and date. the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association
you can use something like this

```ruby
 def add_rental(person, date)
    Rental.new(date, self, person)
  end
end
```